source/simulator/parameters.cc

```diff
@@ -1780,15 +1781,6 @@ namespace aspect
                      ExcMessage ("The advection method 'melt field' can only be selected if melt "
                                  "transport is used in the simulation."));
 
-      if (std::find(compositional_field_methods.begin(), compositional_field_methods.end(), AdvectionFieldMethod::prescribed_field)
-          != compositional_field_methods.end()
-          ||
-          std::find(compositional_field_methods.begin(), compositional_field_methods.end(), AdvectionFieldMethod::prescribed_field_with_diffusion)
-          != compositional_field_methods.end())
-        AssertThrow (!this->use_discontinuous_composition_discretization,
```

Comment:
why do you remove this?

Reply:
I removed the assert, because I use the prevented condition for the benchmarks. We use the prescribed field method to put density into the compositional field, and use discontinuous elements. The assert was only there, because it was not tested.